src/lib/spinel/radio_spinel.hpp

```diff
@@ -1107,7 +1107,7 @@ class RadioSpinel
 
     static constexpr uint32_t kUsPerMs = 1000; ///< Microseconds per millisecond.
     static constexpr uint64_t kTxWaitUs =
-        5000000; ///< Maximum time of waiting for `TransmitDone` event, in microseconds.
+        OPENTHREAD_SPINEL_CONFIG_RCP_TX_WAIT_TIME_SECS * kUsPerMs * 1000; ///< Maximum time of waiting for `TransmitDone` event, in microseconds.
```

Comment:
We use `kUsPerMs` and then hard-coded `* 1000` for the sec to msec conversion part?
Should we define a `kUsPerSec` for this?

Reply:
Thanks Abtin, for reviewing the changes.

Make sense to define `kUsPerSec`, please have a look at the updated changes.